charts/images.yaml

```diff
@@ -0,0 +1,17 @@
+images:
+- name: kubevirt-cloud-controller-manager
+  sourceRepository: github.com/kubevirt/cloud-provider-kubevirt
+  repository: docker.io/mfranczy/kubevirt-cloud-controller-manager
```

Comment:
Could we use an image from an "official" location. Alternatively, could we host this image somewhere under `eu.gcr.io/gardener-project/gardener` if we build it ourselves?

Reply:
Please fix this, you should now have permissions to push into our docker registry.

Reply:
done